kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -1,12 +1,190 @@
 <template>
 
-  <div>
-    <h1>Section Editor</h1>
-    <h2>
-      <KRouterLink appearance="raised-button" :to="{ path: 'select-resources' }">
-        Select resources
-      </KRouterLink>
-    </h2>
+  <div class="section-settings-content">
+    <h5
+      class="section-settings-top-heading"
+    >
+      {{ $tr('sectionSettingsHeading') }}
+    </h5>
+
+    <KGrid>
+      <KGridItem
+        :layout12="{ span: 6 }"
```

Comment:
Using `KGrid` here is a great idea. I think you might want to add some additional layouts though, for smaller screens. It might require a little bit of creativity and input from @tomiwaoLE to decide what the layout should look like on mobile. (On tablet, maybe it should just be a slightly more compact version of the browser view?). This would be applicable not just to this `KGridItem` but for the whole setup. It doesn't have to be a blocker for this PR, but I think it might require some design insight and would be worth discussing sooner rather than later.